tuna/driver_conv.py

```diff
@@ -34,6 +35,7 @@
 from tuna.metadata import DIRECTION, DIR_MAP, CONV_SKIP_ARGS
 from tuna.parsing import get_fd_name, conv_arg_valid, get_fds_from_cmd
 
+LOGGER = setup_logger('driver_conv')
```

Comment:
I think it might be better to setup_logger in the driver_base.py class like this:
LOGGER = setup_logger(a.__class__.__name__)
then we dont have to worry about defining a name for it in each child class(conv, BN, fusions etc).
This line could then be removed

Reply:
You mean to setup the logger as a class variable like worker_interface?

Reply:
oh i just realized its not a class variable, for now, id rather not do that, lets just leave it as it.